### DISCUSSION TITLE
Initial install: Getting "The connection was reset" error in browser

### DISCUSSION THREAD

**Discussion by fivestones** [immich-app/immich#2209] [Q&A]
I did a generic vanilla install, following the docker-compose instructions in the documentation, and after `docker-compose up -d` everything seems to be working, but going to myserver:2283 I just get a "The connection was reset" error.

Trying to view logs doesn't help much either. `docker logs immich_proxy` just gives me "Error response from daemon: configured logging driver does not support reading".  `docker logs immich_web` gives me "Listening on 0.0.0.0:3000". Is there a way to see logs from the immich_proxy container?

Or does anyone else know what I'm doing wrong?

**Comment by fivestones** [comment]
I tried to exec into the immich_proxy container, and got a "Error response from daemon: Container 84347cb1ebe8776d9e692753506802dd0feecff3b01bfb56736de8ec8c0f10e0 is restarting, wait until the container is running" so I guess it's crashing and restarting. Any ideas how I could go about finding out what the problem is?

**Reply by samip5** [reply]
You should look into the logs of the immich_server container and not web.

You can also comment out the logging section for immich_web container to get logs.

**Comment by fivestones** [comment]
I got this working by commenting out the three lines in the .env file:
```
#IMMICH_WEB_URL=http://immich-web::3000
#IMMICH_SERVER_URL=http://immich-server:3001
#IMMICH_MACHINE_LEARNING_URL=http://immich-machine-learning:3003
```
I don't see anything in the installation docs about these. Maybe something should be added? Or they should be commented out by default?

Did I do something wrong, or does a regular vanilla install just work for most people?
